README.rst

```diff
@@ -1,86 +1,91 @@
-parquet-python
-==============
+fastparquet
+===========
 
 .. image:: https://travis-ci.org/jcrobak/parquet-python.svg?branch=master
-    :target: https://travis-ci.org/jcrobak/parquet-python
+    :target: https://github.com/martindurant/fastparquet
 
-parquet-python is a pure-python implementation (currently with only
-read-support) of the `parquet
-format <https://github.com/Parquet/parquet-format>`_. It comes with a
-script for reading parquet files and outputting the data to stdout as
-JSON or TSV (without the overhead of JVM startup). Performance has not
-yet been optimized, but it's useful for debugging and quick viewing of
-data in files.
+fast is a python implementation of the `parquet
+format <https://github.com/Parquet/parquet-format>`_, aiming integrate
+into python-based big data work-flows.
 
 Not all parts of the parquet-format have been implemented yet or tested
-e.g. nested data—see Todos below for a full list. With that said,
-parquet-python is capable of reading all the data files from the
+e.g. see the Todos linked below. With that said,
+fastparquet is capable of reading all the data files from the
 `parquet-compatability <https://github.com/Parquet/parquet-compatibility>`_
 project.
 
-requirements
-============
+Introduction
+------------
 
-parquet-python has been tested on python 2.7, 3.4, and 3.5. It depends
-on ``thrift`` (0.9) and ``python-snappy`` (for snappy compressed files).
+Since the second week of October, this fork of `parquet-python`_ has been
```

Comment:
Recommend having a history section towards the end that includes this information.

I suspect that most people reading the readme will not care about the recent history.